docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -0,0 +1,603 @@
+statements, the connector supports the following features:
+
+* :ref:`sql-data-management`, see also :ref:`delta-lake-write-support`
+* :doc:`/sql/create-schema`, see also :ref:`delta-lake-create-schema`
+* :doc:`/sql/create-table`, see also :ref:`delta-lake-create-table`
+* :doc:`/sql/create-table-as`
+* :doc:`/sql/drop-schema`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/show-create-schema`
+* :doc:`/sql/show-create-table`
+* :ref:`sql-security-operations`, see also :ref:`delta-lake-sql-security`
+
+.. _delta-lake-special-columns:
+
+Special columns
+^^^^^^^^^^^^^^^
+
+In addition to the defined columns, the Delta Lake connector automatically
+exposes metadata in a number of hidden columns in each table. You can use these
+columns in your SQL statements like any other column, e.g., they can be selected
+directly or used in conditional statements.
+
+* ``$path``
+    Full file system path name of the file for this row.
+
+* ``$file_modified_time``
+    Date and time of the last modification of the file for this row.
+
+* ``$file_size``
+    Size of the file for this row.
+
+.. _delta-lake-create-schema:
+
+Creating schemas
+^^^^^^^^^^^^^^^^
+
+The connector supports creating metastore schemas, optionally with a location
+specified.
+
+You can create a schema without a location with the base
+:doc:`/sql/create-schema` statement. Tables in this schema must have a location
+included when you create them. The data files for these tables are not removed
+if the table is dropped::
+
+  CREATE SCHEMA delta.my_schema;
+
+Optionally, a location can be specified. Tables in this schema do not need to
+have a location. By default they are located in a subdirectory under the schema
+location. The data files for tables in this schema using the default location
+are cleaned up if the table is dropped::
+
+  CREATE SCHEMA delta.my_schema WITH (location = 's3://my-bucket/a/path');
+
+.. _delta-lake-create-table:
+
+Creating tables
+^^^^^^^^^^^^^^^
+
+When Delta tables exist in storage, but not in the metastore, Trino can be used
+to register them::
+
+  CREATE TABLE delta.default.my_table (
+    dummy bigint
+  )
+  WITH (
+    location = '...'
+  )
+
+Columns listed in the DDL, such as ``dummy`` in the above example, are ignored.
+The table schema is read from the transaction log, instead. If the schema is
+changed by an external system, Trino automatically uses the new schema.
+
+
+The Delta Lake connector also supports creating tables using the ``CREATE TABLE
+AS`` syntax. There are two table properties available for use in table creation.
+
+The connector automatically writes the initial transaction log entries and
+registers the table in the metastore, when creating a new table. As a result any
+Databricks engine can write to the table::
+
+   CREATE TABLE delta.default.new_table (id bigint, address varchar);
+
+The connector also supports creating tables using the ``CREATE TABLE AS``
+syntax. There are two table properties available for use in table creation.
```

Comment:
Duplicate from earlier .. also what are the properties .. looks like there are three in the examples